[bug] Airtable test-outputs PUT ActionDefinition 103 Error

Sample workflow:
```
{
    "label" : "foo",
    "description" : "",
    "inputs" : [ ],
    "triggers" : [ ],
    "tasks" : [{
        "label" : "Airtable",
        "name" : "airtable_3",
        "parameters" : {
            "baseId" : "appBQVlS69iOKQn4Y",
            "tableId" : "tblBXuPJ7c8mV2zZB",
            "fields" : {
                "fields" : {
                    "Launch Name" : "wer",
                    "Launch date" : "sfd",
                    "Description" : "xvy",
                    "Tasks" : "y",
                    "Features" : "c"
                }
            }
        },
        "type" : "airtable/v1/createRecord",
        "metadata" : {
            "ui" : {
                "dynamicPropertyTypes" : {
                    "fields.fields.Launch Name" : "STRING",
                    "fields.fields.Launch date" : "STRING",
                    "fields.fields.Description" : "STRING",
                    "fields.fields.Tasks" : "STRING",
                    "fields.fields.Features" : "STRING"
                }
            }
        }
    } ]
}
```


Network tab
![Image](https://github.com/user-attachments/assets/12b99275-e5a4-447b-b42b-5fb0845e35ba)

Docker log:
```
com.bytechef.platform.exception.ConfigurationException: com.bytechef.component.exception.ProviderException at com.bytechef.platform.component.util.TokenRefreshHelper.executeSingleConnectionFunction(TokenRefreshHelper.java:96)

Caused by: com.bytechef.component.exception.ProviderException: null at com.bytechef.component.airtable.AirtableComponentHandler.lambda$modifyAction$0(AirtableComponentHandler.java:55)
```

--- a/cli/commands/component/init/openapi/src/main/java/com/bytechef/cli/command/component/init/openapi/ComponentInitOpenApiGenerator.java
+++ b/cli/commands/component/init/openapi/src/main/java/com/bytechef/cli/command/component/init/openapi/ComponentInitOpenApiGenerator.java
@@ -677,6 +677,10 @@ public class ComponentInitOpenApiGenerator {
 
         CodeBlock.Builder metadataBuilder = getMetadataBuilder(operationItem, method, propertiesEntry);
 
+        if (outputEntry != null && outputEntry.isDynamic) {
+            metadataBuilder.add(outputEntry.outputSchemaCodeBlock());
+        }
+
         builder.add(
             """
                 action($S)
@@ -699,15 +703,15 @@ public class ComponentInitOpenApiGenerator {
         CodeBlock outputSchemaCodeBlock = outputEntry == null ? null : outputEntry.outputSchemaCodeBlock();
         CodeBlock sampleOutputCodeBlock = outputEntry == null ? null : outputEntry.sampleOutputCodeBlock();
 
-        if (outputSchemaCodeBlock != null && !outputSchemaCodeBlock.isEmpty()) {
+        if (outputEntry != null && outputEntry.isDynamic) {
+            builder.add(".output()");
+        } else if (outputSchemaCodeBlock != null && !outputSchemaCodeBlock.isEmpty()) {
             if (sampleOutputCodeBlock == null || sampleOutputCodeBlock.isEmpty()) {
                 builder.add(".output(outputSchema($L))", outputSchemaCodeBlock);
             } else {
                 builder.add(".output(outputSchema($L), sampleOutput($L))", outputSchemaCodeBlock,
                     sampleOutputCodeBlock);
             }
-        } else if (outputEntry != null && outputEntry.isDynamic()) {
-            builder.add(".output()");
         }
 
         return builder.build();
@@ -1358,6 +1362,7 @@ public class ComponentInitOpenApiGenerator {
                 propertiesEntry.bodyContentType,
                 propertiesEntry.mimeType);
         }
+
         return metadataBuilder;
     }
 
@@ -1439,7 +1444,40 @@ public class ComponentInitOpenApiGenerator {
         }
 
         if (apiResponse != null) {
-            if (apiResponse.getContent() != null) {
+            if (apiResponse.getExtensions() != null) {
+                Map<String, Object> extensions = apiResponse.getExtensions();
+
+                if (extensions != null) {
+                    Object dynamicOutput = extensions.get("x-dynamic-output");
+
+                    if (dynamicOutput.equals(true)) {
+
+                        Content content = apiResponse.getContent();
+
+                        Set<Map.Entry<String, MediaType>> entries = content.entrySet();
+
+                        if (!entries.isEmpty()) {
+                            String mimeType = getMimeType(entries);
+                            String responseType = switch (mimeType) {
+                                case "application/json" -> "JSON";
+                                case "application/xml" -> "XML";
+                                case "application/octet-stream" -> "BINARY";
+                                default -> "TEXT";
+                            };
+
+                            CodeBlock.Builder codeBlock = CodeBlock.builder();
+
+                            codeBlock.add(
+                                """
+                                    ,"responseType", ResponseType.$L
+                                    """,
+                                responseType);
+
+                            outputEntry = new OutputEntry(codeBlock.build(), null, true);
+                        }
+                    }
+                }
+            } else if (apiResponse.getExtensions() == null && apiResponse.getContent() != null) {
                 Content content = apiResponse.getContent();
 
                 Set<Map.Entry<String, MediaType>> entries = content.entrySet();
@@ -1453,16 +1491,6 @@ public class ComponentInitOpenApiGenerator {
                         getOutputSchemaCodeBlock(mimeType, mediaType),
                         getSampleOutputCodeBlock(mediaType.getExample()),
                         false);
-                }
-            } else if (apiResponse.getContent() == null && apiResponse.getExtensions() != null) {
-                Map<String, Object> extensions = apiResponse.getExtensions();
-
-                if (extensions != null) {
-                    Object dynamicOutput = extensions.get("x-dynamic-output");
-
-                    if (dynamicOutput.equals(true)) {
-                        outputEntry = new OutputEntry(null, null, true);
-                    }
                 }
             }
         }

--- a/server/libs/modules/components/airtable/openapi.yaml
+++ b/server/libs/modules/components/airtable/openapi.yaml
@@ -49,6 +49,10 @@ paths:
         "200":
           description: "Successful operation"
           x-dynamic-output: true
+          content:
+            application/json:
+              schema:
+                type: "object"
   /{baseId}/{tableId}/{recordId}:
     delete:
       summary: "Delete Record"
@@ -138,6 +142,10 @@ paths:
         200:
           description: "Successful operation"
           x-dynamic-output: true
+          content:
+            application/json:
+              schema:
+                type: "object"
 components:
   securitySchemes:
     api_key:

--- a/server/libs/modules/components/airtable/src/main/java/com/bytechef/component/airtable/action/AirtableCreateRecordAction.java
+++ b/server/libs/modules/components/airtable/src/main/java/com/bytechef/component/airtable/action/AirtableCreateRecordAction.java
@@ -21,6 +21,7 @@ import static com.bytechef.component.definition.ComponentDsl.action;
 import static com.bytechef.component.definition.ComponentDsl.dynamicProperties;
 import static com.bytechef.component.definition.ComponentDsl.string;
 import static com.bytechef.component.definition.Context.Http.BodyContentType;
+import static com.bytechef.component.definition.Context.Http.ResponseType;
 
 import com.bytechef.component.airtable.util.AirtableUtils;
 import com.bytechef.component.definition.ComponentDsl;
@@ -40,7 +41,8 @@ public class AirtableCreateRecordAction {
         .metadata(
             Map.of(
                 "method", "POST",
-                "path", "/{baseId}/{tableId}", "bodyContentType", BodyContentType.JSON, "mimeType", "application/json"
+                "path", "/{baseId}/{tableId}", "bodyContentType", BodyContentType.JSON, "mimeType", "application/json",
+                "responseType", ResponseType.JSON
 
             ))
         .properties(string("baseId").label("Base ID")

--- a/server/libs/modules/components/airtable/src/main/java/com/bytechef/component/airtable/action/AirtableGetRecordAction.java
+++ b/server/libs/modules/components/airtable/src/main/java/com/bytechef/component/airtable/action/AirtableGetRecordAction.java
@@ -19,6 +19,7 @@ package com.bytechef.component.airtable.action;
 import static com.bytechef.component.OpenApiComponentHandler.PropertyType;
 import static com.bytechef.component.definition.ComponentDsl.action;
 import static com.bytechef.component.definition.ComponentDsl.string;
+import static com.bytechef.component.definition.Context.Http.ResponseType;
 
 import com.bytechef.component.airtable.util.AirtableUtils;
 import com.bytechef.component.definition.ComponentDsl;
@@ -37,7 +38,7 @@ public class AirtableGetRecordAction {
         .metadata(
             Map.of(
                 "method", "GET",
-                "path", "/{baseId}/{tableId}/{recordId}"
+                "path", "/{baseId}/{tableId}/{recordId}", "responseType", ResponseType.JSON
 
             ))
         .properties(string("baseId").label("Base ID")

--- a/server/libs/modules/components/airtable/src/test/resources/definition/airtable_v1.json
+++ b/server/libs/modules/components/airtable/src/test/resources/definition/airtable_v1.json
@@ -6,6 +6,10 @@
     "help" : null,
     "metadata" : {
       "path" : "/{baseId}/{tableId}",
+      "responseType" : {
+        "contentType" : "application/json",
+        "type" : "JSON"
+      },
       "mimeType" : "application/json",
       "method" : "POST",
       "bodyContentType" : "JSON"
@@ -341,7 +345,11 @@
     "help" : null,
     "metadata" : {
       "method" : "GET",
-      "path" : "/{baseId}/{tableId}/{recordId}"
+      "path" : "/{baseId}/{tableId}/{recordId}",
+      "responseType" : {
+        "contentType" : "application/json",
+        "type" : "JSON"
+      }
     },
     "name" : "getRecord",
     "outputDefinition" : {

--- a/server/libs/modules/components/wolfram-alpha/wolfram-alpha-full-results/openapi.yaml
+++ b/server/libs/modules/components/wolfram-alpha/wolfram-alpha-full-results/openapi.yaml
@@ -235,6 +235,10 @@ paths:
         200:
           description: "Full result."
           x-dynamic-output: true
+          content:
+            application/xml:
+              schema:
+                type: "object"
 components:
   securitySchemes:
     ApiKeyQuery:

--- a/server/libs/modules/components/wolfram-alpha/wolfram-alpha-full-results/src/main/java/com/bytechef/component/wolfram/alpha/full/results/action/WolframAlphaFullResultsGetFullResultAction.java
+++ b/server/libs/modules/components/wolfram-alpha/wolfram-alpha-full-results/src/main/java/com/bytechef/component/wolfram/alpha/full/results/action/WolframAlphaFullResultsGetFullResultAction.java
@@ -22,6 +22,7 @@ import static com.bytechef.component.definition.ComponentDsl.array;
 import static com.bytechef.component.definition.ComponentDsl.bool;
 import static com.bytechef.component.definition.ComponentDsl.option;
 import static com.bytechef.component.definition.ComponentDsl.string;
+import static com.bytechef.component.definition.Context.Http.ResponseType;
 
 import com.bytechef.component.definition.ComponentDsl;
 import java.util.Map;
@@ -38,7 +39,7 @@ public class WolframAlphaFullResultsGetFullResultAction {
         .metadata(
             Map.of(
                 "method", "GET",
-                "path", "/query"
+                "path", "/query", "responseType", ResponseType.XML
 
             ))
         .properties(string("input").label("Query")

--- a/server/libs/modules/components/wolfram-alpha/wolfram-alpha-full-results/src/test/resources/definition/wolfram-alpha-full-results_v1.json
+++ b/server/libs/modules/components/wolfram-alpha/wolfram-alpha-full-results/src/test/resources/definition/wolfram-alpha-full-results_v1.json
@@ -6,7 +6,11 @@
     "help" : null,
     "metadata" : {
       "method" : "GET",
-      "path" : "/query"
+      "path" : "/query",
+      "responseType" : {
+        "contentType" : "application/xml",
+        "type" : "XML"
+      }
     },
     "name" : "getFullResult",
     "outputDefinition" : {

--- a/server/libs/platform/platform-component/platform-component-api/src/main/java/com/bytechef/platform/component/util/OpenApiClientUtils.java
+++ b/server/libs/platform/platform-component/platform-component-api/src/main/java/com/bytechef/platform/component/util/OpenApiClientUtils.java
@@ -60,7 +60,7 @@ public class OpenApiClientUtils {
         BaseValueProperty<?> outputSchema = outputDefinition == null ? null : outputDefinition.getOutputSchema();
 
         ResponseType responseType = outputSchema == null
-            ? null
+            ? MapUtils.get(metadata, "responseType", ResponseType.class)
             : MapUtils.get(outputSchema.getMetadata(), "responseType", ResponseType.class);
 
         Response response = context

